sudoers/map.jinja

```diff
@@ -1,14 +1,30 @@
 {% set sudoers = salt['grains.filter_by']({
-    'Debian':  {'pkg': 'sudo'},
-    'Ubuntu':  {'pkg': 'sudo'},
-    'CentOS':  {'pkg': 'sudo'},
-    'Fedora':  {'pkg': 'sudo'},
-    'RedHat':  {'pkg': 'sudo'},
-    'Amazon':  {'pkg': 'sudo'},
-    'Gentoo':  {'pkg': 'app-admin/sudo'},
-    'Mint':    {'pkg': 'sudo'},
-    'Arch':    {'pkg': 'sudo'},
-    'FreeBSD': {'pkg': 'sudo',
+    'defaults': {
+                'lookup': {
+                    'pkg': 'sudo',
+                    },
+                'included': False,
+                'includedir': None,
+                },
+    'Debian': {
+                'lookup': {
+                  'pkg': 'sudo',
+                },
+                'defaults': {
+                  'generic': [
+                    'env_reset',
+                    'mail_badpass',
+                    'secure_path="/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin"',
+                    ],
+                },
+                'users':  {'root': ['ALL=(ALL:ALL) ALL']},
+                'groups': {'sudo': ['ALL=(ALL:ALL) ALL']},
+                'includedir': None,
+               },
+    'Gentoo':  { 'lookup':{'pkg': 'app-admin/sudo'}},
+    'FreeBSD': { 'lookup':{'pkg': 'sudo',
                 'config-path': '/usr/local/etc',
                 'group': 'wheel'},
-}, merge=salt['pillar.get']('sudoers:lookup')) %}
+               },
+}, grain='os_family', merge=salt['pillar.get']('sudoers'), default='defaults') %}
```

Comment:
By specifying a default dict, anything not matching the listed os_family's will receive the values specified in `defaults: {}`.

Also, switched to os_family to match Debian, Mint and Ubuntu.